scrapers/mt/bills.py

```diff
@@ -133,7 +133,7 @@ def parse_bill(self, bill_url, list_sponsor, session):
         bill = None
         doc = self.lxmlize(bill_url)
 
-        bill, votes = self.parse_bill_status_page(bill_url, doc, list_sponsor, session)
+        bill, votes = self.parse_bill_status_page(bill_url, doc, session)
```

Comment:
Given the comment on line 132 I'm missing something here, but the function doesn't accept a `list_sponsor` as is

Reply:
Ah nvm I see, updating